packages/thumbprint-scss/mixins/type.scss

```diff
@@ -88,74 +88,74 @@
 // DEPRECATED
 
 @mixin tp-heading-1($important: null) {
-    font-size: $tp-font__heading__1__size $important;
-    line-height: $tp-font__heading__1__line-height $important;
-    font-weight: $tp-font__heading__1__weight $important;
+    font-size: 32px $important;
+    line-height: 1.3 $important;
+    font-weight: 500 $important;
```

Comment:
`500` can be `400` since our Mark font doesn't have that weight and is loading that now anyway.

Reply:
Oh, right. Forgot about that. Will fix.